plugins/discourse-local-dates/test/javascripts/lib/local-date-builder-test.js.es6

```diff
@@ -415,7 +415,7 @@ test("previews", function (assert) {
 
   freezeTime({ date: "2020-03-22", timezone: PARIS }, () => {
     assert.buildsCorrectDate(
-      { timezone: PARIS, isplayedTimezone: PARIS },
+      { timezone: PARIS, displayedTimezone: PARIS },
```

Comment:
?!

Reply:
Looks like a blatantly broken test to me.